src/pages/schedule.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import Layout from '../components/layout';
+import Schedule from '../components/SchedulePage/Schedule';
+
+const SchedulePage = () => {
+	return <Layout>
+		<Schedule/>
```

Comment:
You should add a `<SEO />` element, like https://github.com/uclaacm/hoth.uclaacm.com/blob/e4e5b087ff8cf58aa9b222778b33121c6501cf3a/src/pages/index.js#L12.